web/cobrands/sass/_base.scss

```diff
@@ -2373,26 +2373,13 @@ label .muted {
       }
     }
   }
-  a#geolocate_link {
-    @include inline-block;
-    vertical-align:top;
-    background:#1a1a1a;
-    color:#C8C8C8;
-    padding:0.5em;
-    font-family: $meta-font;
-    font-size: 0.8125em;
-    @include border-radius(0 0 0.25em 0.25em);
-    &:hover {
-      text-decoration:none;
-      background:#2a2a2a;
-    }
-  }
-  a#geolocate_link.loading {
-      background: #1a1a1a url("/cobrands/fixmystreet/images/spinner-black.gif") flip(100%,0) 50% no-repeat;
```

Comment:
Can remove the GIF, I think this was its only use.

Reply:
I’m wondering whether to just leave it there, in case a future cobrand wants to use it? Given we’ve already got half a dozen other `spinner-*.gif` colour options there. Or is that just messy?

Reply:
Meh, don't mind.